fatiando/gravmag/transform.py

```diff
@@ -141,6 +141,10 @@ def upcontinue(x, y, data, shape, height):
 
     .. note:: x, y, z and height should be in meters.
 
+    .. note:: It is not possible to get the FFT of a masked grid. The default
+    `gridder.interp()` call using minimum curvature will not be suitable. Use
+    `extrapolate=True` or `algorithm='nearest'`.
+
```

Comment:
@mtb-za I tried compiling the docs on my machine and this is giving errors. For multiline `..note::`s you'll need to break the line and ident the whole thing, [like this](https://github.com/fatiando/fatiando/blob/master/fatiando/gravmag/transform.py#L70):

        .. warning::

            The magnetization direction of the anomaly source is crucial to the
            reduction-to-the-pole.
            **Wrong values of *sinc* and *sdec* will lead to a wrong reduction.**